pwndbg/aglib/arch.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+import pwndbg
+
+# We will optimize this module in the future, by having it work in the same
+# way the `gdblib` version of it works, and that will come at the same
+# time this module gets expanded to have the full feature set of its `gdlib`
+# coutnerpart. For now, though, this should be good enough.
+
+
+def __getattr__(name):
+    if name == "endian":
```

Comment:
Eh, its really not good and will be super slow :<

We need to compute all this bs once for process :p

Reply:
I agree. But like I explained in the comment above, this is a fairly temporary solution. Doing this the right way - i.e. the way it's done in `gdblib` - is not really possible until LLDB has support for the event system. That shouldn't be long at all, though.